routes/HomePage.vue

```diff
@@ -12,19 +12,19 @@
 </template>
 
 <script>
-import {computedAsync} from '@vueuse/core';
-import {computed, ref, toRef, watch} from 'vue';
-import Credentials from '../components/Credentials.vue';
 import {
   ageCredentialHelpers,
   getCredentialStore,
   profileManager
 } from '@bedrock/web-wallet';
+import {computed, ref, toRef, watch} from 'vue';
+import {computedAsync} from '@vueuse/core';
+import Credentials from '../components/Credentials.vue';
 
 export default {
   name: 'HomePage',
   components: {
-    Credentials
+    Credentials,
```

Comment:
Unnecessary extra addition of trailing comma.

```suggestion
    Credentials
```